definitions/npm/react-redux_v5.x.x/flow_v0.89.x-/test_connectInfer.js

```diff
@@ -75,45 +75,6 @@ function doesNotRequireDefinedComponentToTypeCheck5case() {
   connect(mapStateToProps, mapDispatchToProps, mergeProps)(Component);
 }
 
-function testExactProps() {
```

Comment:
This test now fails with the new change; types can no longer be inferred for exact props.